cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -149,10 +149,23 @@ write_files:
       [ -f /etc/profile.d/proxy.sh ] && . /etc/profile.d/proxy.sh
       custom_cookbook=${CustomChefCookbook}
       export _region=${AWS::Region}
+
+      # The default curl setting will timeout the connect attempt at 5 minutes, so
+      # having the retry count set to 3 will result in 15m to failure if there is
+      # no internet connectivity from the compute node.
+      cookbook_url_retry_strategy=--retry 3
       s3_url=${AWS::URLSuffix}
       if [ "${!custom_cookbook}" != "NONE" ]; then
         if [[ "${!custom_cookbook}" =~ ^s3://([^/]*)(.*) ]]; then
-          bucket_region=$(aws s3api get-bucket-location --bucket ${!BASH_REMATCH[1]} | jq -r '.LocationConstraint')
+          export AWS_RETRY_MODE=standard
+          export AWS_MAX_ATTEMPTS=2
```

Comment:
With export `AWS_MAX_ATTEMPTS` variable here, is this also going to impact the numbers of retries for AWS CLI commands to use somewhere else? Do we want to change the retries to 2 for all the AWS CLI commands? Or do we need to unset AWS_MAX_ATTEMPTS after running `aws s3api` command?